### PR TITLE
Docker tag v1

### PR DESCRIPTION
**Pull request by zionyx**
* Added simple docker tag and its tests

### CODE REVIEW

**Comment by zionyx** [comment]
Rebased to master.

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+[![Build status](https://api.travis-ci.com/3shapeAS/dockerbuild-pwsh.svg?branch=master)](https://travis-ci.com/3shapeAS/dockerbuild-pwsh)
+
 # dockerbuild-pwsh
 Powershell Core script to build and test Docker aimges.
```
Comment (rasmusjelsgaard) [inline README.md]:
```suggestion
Powershell Core script to build and test Docker images.
```

File: README.md [hunk 2]
```diff
@@ -1,3 +1,5 @@
+[![Build status](https://api.travis-ci.com/3shapeAS/dockerbuild-pwsh.svg?branch=master)](https://travis-ci.com/3shapeAS/dockerbuild-pwsh)
```
Comment (rasmusjelsgaard) [inline README.md]: 🌞

File: Tests/Invoke-DockerTag.Tests.ps1 [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+        BeforeEach {
+            Initialize-MockReg
+        }
+
+        $code = {
+            Write-Debug $Command
+            StoreMockValue -Key "mock" -Value $Command
+        }
+
+        It 'tags docker image with image name only' {
```
Comment (rasmusjelsgaard) [inline Tests/Invoke-DockerTag.Tests.ps1]: I like these tests. They are well written and concise.

File: Public/Invoke-DockerTag.ps1 [hunk 4]
```diff
@@ -0,0 +1,20 @@
+#Requires -PSEdition Core -Version 6
+
+function Invoke-DockerTag {
+    [CmdletBinding()]
+    param (
+        [Parameter(mandatory=$true)]
+        [String]
+        $SourceImage,
+        [ValidateNotNullOrEmpty()]
+        [String]
```
Comment (rasmusjelsgaard) [inline Public/Invoke-DockerTag.ps1]: I am not sure this even works as expected. I think you should add test cases where you try setting them to null or an empty string to check.

File: Tests/Invoke-DockerTag.Tests.ps1 [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+            $result = GetMockValue -Key "mock"
+            Write-Debug $result
+            $result | Should -BeLikeExactly "docker tag lalaland:latest lololand:latest"
+        }
+
+        It 'tags docker image with image name with latest tag' {
+
+            Mock -CommandName "Invoke-Command" $code -Verifiable -ModuleName $script:moduleName
+
+            Invoke-DockerTag -SourceImage 'lalaland' -SourceTag 'latest' -TargetImage 'lololand' -TargetTag 'latest'
```
Comment (rasmusjelsgaard) [inline Tests/Invoke-DockerTag.Tests.ps1]: Since 'latest' is the default value it would make the test more valuable if you try and pass something else than 'latest' to verify it gets set correctly.

File: README.md (hunk 2 above)
Comment (zionyx) [reply README.md]: 🌻

File: Public/Invoke-DockerTag.ps1 (hunk 4 above)
Comment (zionyx) [reply Public/Invoke-DockerTag.ps1]: 🌞

File: Tests/Invoke-DockerTag.Tests.ps1 (hunk 5 above)
Comment (zionyx) [reply Tests/Invoke-DockerTag.Tests.ps1]: Fixed! 🌞

**Comment by zionyx** [comment]
Rebased to master.
